### ISSUE TITLE
Hi! I wanna add String read when Input specific_size, I will submit a PR

### ISSUE THREAD

**Issue by weykon** [dnlmlr/cargo-clean-all#6]
I am user of this crate,  I confuse at unit of size input when I using.

so I wanna fix it complete at this point.

could I submit a PR for this?

@dnlmlr

**Comment by dnlmlr** [comment]
The size input is currenty simply in bytes, which I agree is pretty bad. This is fixed (you can specify the unit e.g. 20MB) on the dev branch. 

There is some weirdness going on with interactive dialogs from the dialoguer crate, which is the reason why it is not yet merged. This seems to be a general problem with interactive terminal dialogs and is probably not going to be fixed upstream. So I will try to apply a workaround that makes this issue more unlikely and also make interactive mode not the default.

Long story short, the new version with this fix will release this weekend. Until then you can specify the size in bytes. So 10MB would be 10000000.

**Comment by dnlmlr** [comment]
I just published the new version `0.5.0` which incorporates the ability to specify size units with the `--keep-size` / `-s` parameter. 
You can install it as always with `cargo install cargo-clean-all`.

I'll close this issue, as your problem should be solved with the new version. If there are any other problems, feel free to open a new issue, or to reopen this one if there are any problems with the size specification

**Comment by weykon** [comment]
> I just published the new version `0.5.0` which incorporates the ability to specify size units with the `--keep-size` / `-s` parameter. You can install it as always with `cargo install cargo-clean-all`.
> 
> I'll close this issue, as your problem should be solved with the new version. If there are any other problems, feel free to open a new issue, or to reopen this one if there are any problems with the size specification

Oh, you work quickly. I'm late because timezone.
Thanks!
